gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/handler/login/LoginCallbackFailureHandler.java

```diff
@@ -283,7 +285,15 @@ private void redirectToLoginPage(MultiMap originalParams,
         }
         params.set(Parameters.CLIENT_ID, client.getClientId());
         params.set(ConstantKeys.ERROR_PARAM_KEY, "social_authentication_failed");
-        params.set(ConstantKeys.ERROR_DESCRIPTION_PARAM_KEY, encodeURIComponent(throwable.getCause() != null ? throwable.getCause().getMessage() : throwable.getMessage()));
+        String errorDescription = (throwable.getCause() != null ? throwable.getCause().getMessage() : throwable.getMessage());
+        params.set(ConstantKeys.ERROR_DESCRIPTION_PARAM_KEY, errorDescription);
```

Comment:
`errorDescription` used to be URI-encoded via `encodeURIComponent`. We do use it as a URI param, so I think we should keep encoding it.